SMSEagleIOTBridge/Readme_Prepare_Eagle.md

```diff
@@ -117,167 +96,128 @@ should give
+The environment variables are configured using a so called Drop-In file.  The following describes how to create and fill it directly on the SMSEagle.
 
-As for the method to create it directly on the SMSEagle:
+2. Create the following folder
 ```
 mkdir /etc/systemd/system/nyss-iot-bridge.service.d
-```
-and a file 
-```
+``` 
+3. Create the file
+``` 
 touch /etc/systemd/system/nyss-iot-bridge.service.d/override.conf
 ```
-The file takes the environment variables and should look e.g. as such (make sure that if you just copy it from here, and paste using vi, you are not missing the beginning of the first line. The buffer seems to be too small.):
+
+4. Enter text editor
+
+```
+nano /etc/systemd/system/nyss-iot-bridge.service.d/override.conf
+``` 
+
+5. Fill in environment variables
+
+The file takes the environment variables and should look e.g. as such (make sure that if you just copy it from here, and paste using nano, you are not missing the beginning of the first line. The buffer seems to be too small.):
+
 ```
 [Service]
 Environment="IOT_HUB_CONNECTIONSTRING=HostName=iothuburl;DeviceId=somedevice;SharedAccessKey=somerandomkey"
 Environment="SMSEAGLE_USERNAME=usernameyoucreatedbefore"
 Environment="SMSEAGLE_PWD=pwdyoucreatedbefore"
 ```
+The connection string is what you configured in step 1.4.4
+The username and password is what you configured in step 1.3.3
 
-You can get the IOT_HUB_CONNECTIONSTRING variable from the Azure IOT Hub (Primary Connection String). The creation of the Eagle in Azure IOT Hub is described in the respective Readme file coming with the Nyss codebase. If that is not available, refer to the Microsoft tutorials on how to create an IOT device. 
-The IOT_HUB_CONNECTIONSTRING is what is refered to as the Primary Connection String in Azure IOT Hub.
-
-### 1.2.6. Start service & Wrap up
+## 2.4 Start service & Wrap up
 
-After configuring the above, you should run the following commands:
+1. Reload and restart service file
 
 ```
 systemctl daemon-reload
 systemctl enable nyss-iot-bridge.service
 systemctl start nyss-iot-bridge.service
 reboot
 ```
-
-You can check the status of the service using:
+2. Check status of service file
 ```
 systemctl status nyss-iot-bridge.service
 ```
-
-The python scripts logs to:
-
-```
-/var/log/iot-bridge-log.log
-```
-
-You can check that log file using e.g.
+3. You can also check the python script log
 ```
-vi /var/log/iot-bridge-log.log
+nano /var/log/iot-bridge-log.log
 ```
 
-Shift + G brings you to the bottom of the file in vim.
-
-### 1.2.7. Check its working and common failures
+# 3. Status checks and common failures
 
-After reboot the following command should:
-```
+## 3.1 Check the service is running
+After reboot, run the following command
+´´´
 systemctl status nyss-iot-bridge.service
-```
-should show something like the following:
-```
+´´´
+The message should show something like this
+´´´
  nyss-iot-bridge.service - This is the service for the iot bridge to azure
```

Comment:
The formatting here is a bit hard to read, maybe add a screenshot of the terminal output?